From 95f5183700cad0c713bc4f5757555b45ce173ed8 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Sun, 24 Jun 2012 21:02:04 -0400
Subject: [PATCH] Stop using obsolete $(mkdir_p) for automake-1.12
 compatibility

$(mkdir_p) was declared obsolete in automake-1.10, deprecated in 1.12,
and removed in 1.13.

https://bugs.freedesktop.org/show_bug.cgi?id=51392
---
 src/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index ede1fc3..154c56e 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -57,7 +57,7 @@ _generated/errors.py: $(tools_dir)/python-errors-generator.xsl $(wildcard $(spec
 	    $(spec_dir)/all.xml
 
 _generated/__init__.py:
-	$(AM_V_GEN)$(mkdir_p) $(dir $@)
+	$(AM_V_GEN)$(MKDIR_P) $(dir $@)
 	@echo "# Placeholder for package" > $@
 
 _generated/%.py: $(tools_dir)/spec-to-python.xsl $(spec_dir)/%.xml
-- 
1.7.8.6

